Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h

```diff
@@ -568,21 +568,33 @@ SWIFT_CLASS_NAMED("EntitlementInfos")
 /// Dictionary of active <code>EntitlementInfo</code> objects keyed by their identifiers.
 /// warning:
 /// this is equivalent to <code>activeInAnyEnvironment</code>
-/// seealso:
-/// <code>activeInCurrentEnvironment</code>
+/// <h4>Related Symbols</h4>
```

Comment:
doesn't really matter but i'm assuming this means main isn't fully merged into this branch..?

Reply:
Hmm what do you mean? This should have everything that's in `main`. Did I miss something?

Reply:
oh or the seealso fix went into this branch** nope all good

Reply:
Yup!